src/lib/generator/index.js

```diff
@@ -1,2 +1,20 @@
+import { resolve, join } from 'path';
+
+const INPUT_DIR = 'src/files/input/';
+const OUTPUT_DIR = 'src/files/output/';
+const icdDescriptionsFilePath = resolve(join(INPUT_DIR, 'icd_descriptions.txt'));
+const hccMappingsFilePath = resolve(join('hcc_mappings.csv'));
+const icdCodesJsonFilePath = resolve(join(OUTPUT_DIR, 'icd_codes.json'));
+const icdCodesCsvFilePath = resolve(join(OUTPUT_DIR, 'icd_codes.csv'));
+
+export {
```

Comment:
I don't know if this is really necessary.. at first I did it so that I could test the functions without hardcoded urls, but then I realized I should just extract the part I wanted to test to a separate function